test/metabase/db/force_migration_test.clj

```diff
@@ -28,8 +28,8 @@
               {:name "Egypt"
                :period "3100BC"}))
 
-;; This test tests 4 things:
-;; - It forces release the locks if there is one
+;; This test tests 4 (on h2) or 3 (on others) things:
+;; - It forces release the locks if there is one (only on h2, others use a session lock that can't be forced)
```

Comment:
Since only h2 can release locks now, it seems like the `force-release-locks!` function should throw a user-helpful error like "Database X does not support forcing migration locks. The lock is always automatically removed when the process performing the migration completes" for any non-h2 databases?

Reply:
Done